electionleaflets/templates/base.html

```diff
@@ -1,112 +1,77 @@
-    {% block extra_site_css %}
-      {% stylesheet 'styles' %}
-    {% endblock extra_site_css %}
-    {% block site_meta %}
-      <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-      <meta name="mobile-web-app-capable" content="yes" />
-      <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=1">
-      <meta name="keywords" lang="en" content="election leaflets" />
-      <meta name="description" lang="en" content="Real-time election leaflet monitoring" />
-      <title>{% block title %}Real-time #ElectionLeaflet monitoring{% endblock %} | ElectionLeaflets.org</title>
-    {% endblock site_meta %}
-    {% block og_tags %}{% endblock og_tags %}
-  </head>
-  <body class="ds-scope" style="margin: 0">
-    <noscript>
-        <div class="ds-status-error">
-            JavaScript is disabled in your browser. Please enable JavaScript to upload leaflets.
-        </div>
-    </noscript>
-  
-    <div class="ds-page">
-      <p><a class="ds-skip-link" href="#main">skip to content</a></p>
-      <header class="ds-header">
-        <a class="ds-logo" href="/">
-          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
-          <span>Election Leaflets</span>
-        </a>
-        <nav>
-          <ul>
-            <li>
-              <a href="{% url "home" %}">Home</a>
-            </li>
-            {% if request.user.is_authenticated %}
-              <li>
-                <a href="{% url "leaflets_without_candidates" %}">Tag</a>
-              </li>
-              <li>
-                <a href="{% url "moderate" %}">Mod Queue</a>
-              </li>
-            {% endif %}
-            <li>
-              <a href="{% url "leaflets" %}">Latest leaflets</a>
-            </li>
-            <li>
-              <a href="{% url "upload_leaflet" %}">Add a leaflet</a>
-            </li>
-          </ul>
-        </nav>
-      </header>
-      <main id="main" tabindex="-1" class="ds-stack" {% if full_width %}style="max-width:100%"{% endif %}>
+
+{% block extra_site_css %}
+  {% stylesheet 'styles' %}
+{% endblock extra_site_css %}
+
+{% block base_title %}{% block page_title %}Real-time #ElectionLeaflet monitoring{% endblock page_title %} |ElectionLeaflets.org {% endblock base_title %}
+
+{% block page_meta %}
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta name="mobile-web-app-capable" content="yes" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=1">
+  <meta name="keywords" lang="en" content="election leaflets" />
+  <meta name="description" lang="en" content="Real-time election leaflet monitoring" />
+{% endblock page_meta %}
+
+{% block og_tags %}{% endblock og_tags %}
+<body class="ds-scope" style="margin: 0">
+  <noscript>
+      <div class="ds-status-error">
+          JavaScript is disabled in your browser. Please enable JavaScript to upload leaflets.
+      </div>
+  </noscript>
```

Comment:
These are from the original `base.html` and I wasn't sure what to do with them.  Can they be removed or do they need to be put in a block from `dc_base.html`?